docs/toc.md

```diff
@@ -32,84 +32,84 @@
+        - [API - EntityBuilder]({{urlRoot}}/content/api-entity-builder)
+    - Commands
+        - [World and component command requests and responses]({{urlRoot}}/content/world-component-commands-requests-responses)
 - <h3> + Tools </h3>
     - [Snapshots]({{urlRoot}}/content/snapshots)
-    - <h4>Code generation:</h4>
-        - [* The code generator]({{urlRoot}}/content/code-generator)
-        - [* ECS component generation]({{urlRoot}}/content/ecs/component-generation)
+    - Code generation
+        - [The code generator]({{urlRoot}}/content/code-generator)
+        - [ECS component generation]({{urlRoot}}/content/ecs/component-generation)
     - [Logging]({{urlRoot}}/content/ecs/logging)
 - <h3> + MonoBehaviour workflow</h3>
     - [Which workflow?]({{urlRoot}}/content/intro-workflows-spatialos-entities)
-    - <h4>Create workers:</h4>
-        - [* WorkerConnector]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector)
-        - [* API - WorkerConnector]({{urlRoot}}/content/gameobject/api-workerconnector)
-    - <h4>Link to GameObjects:</h4>
-        - [* SpatialOS entities as GameObjects]({{urlRoot}}/content/gameobject/linking-spatialos-entities)
-        - [* Workers as GameObjects]({{urlRoot}}/content/gameobject/linking-workers-gameobjects)
-    - <h4>MonoBehaviours with SpatialOS:</h4>
-        - [* Interact with SpatialOS using MonoBehaviours]({{urlRoot}}/content/gameobject/interact-spatialos-monobehaviours)
-        - [* Readers and writers]({{urlRoot}}/content/gameobject/readers-writers)
-        - [* Readers and writers: SpatialOS component data]({{urlRoot}}/content/gameobject/reading-and-writing-component-data)
-        - [* Readers and writers: Events]({{urlRoot}}/content/gameobject/sending-receiving-events)
-        - [* Commands: SpatialOS Component commands]({{urlRoot}}/content/gameobject/sending-receiving-commands)
-        - [* Commands: World commands]({{urlRoot}}/content/gameobject/world-commands)
+    - Create workers
+        - [WorkerConnector]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector)
+        - [API - WorkerConnector]({{urlRoot}}/content/gameobject/api-workerconnector)
+    - Link to GameObjects
+        - [SpatialOS entities as GameObjects]({{urlRoot}}/content/gameobject/linking-spatialos-entities)
+        - [Workers as GameObjects]({{urlRoot}}/content/gameobject/linking-workers-gameobjects)
+    - MonoBehaviours with SpatialOS
+        - [Interact with SpatialOS using MonoBehaviours]({{urlRoot}}/content/gameobject/interact-spatialos-monobehaviours)
+        - [Readers and writers]({{urlRoot}}/content/gameobject/readers-writers)
+        - [Readers and writers: SpatialOS component data]({{urlRoot}}/content/gameobject/reading-and-writing-component-data)
+        - [Readers and writers: Events]({{urlRoot}}/content/gameobject/sending-receiving-events)
+        - [Commands: SpatialOS Component commands]({{urlRoot}}/content/gameobject/sending-receiving-commands)
```

Comment:
Component -> component?

Reply:
👍

Reply:
Fixed.

Reply:
Might be being dim, but it looks like "Component" is still capitalised here?

Reply:
Apparently, you have to SAVE the file before committing. Who knew?